deploy/docker/cp-tests/e2e-ldap/krb5.conf

```diff
@@ -0,0 +1,99 @@
+[libdefaults]
+	default_realm = CPPL.COM
+
+# The following krb5.conf variables are only for MIT Kerberos.
+	kdc_timesync = 1
+	ccache_type = 4
+	forwardable = true
+	proxiable = true
+
+# The following encryption type specification will be used by MIT Kerberos
+# if uncommented.  In general, the defaults in the MIT Kerberos code are
+# correct and overriding these specifications only serves to disable new
+# encryption types as they are added, creating interoperability problems.
+#
+# The only time when you might need to uncomment these lines and change
+# the enctypes is if you have local software that will break on ticket
+# caches containing ticket encryption types it doesn't know about (such as
+# old versions of Sun Java).
+
+#	default_tgs_enctypes = des3-hmac-sha1
+#	default_tkt_enctypes = des3-hmac-sha1
+#	permitted_enctypes = des3-hmac-sha1
+
+# The following libdefaults parameters are only for Heimdal Kerberos.
+	fcc-mit-ticketflags = true
+
+[realms]
+	CPPL.COM = {
+		kdc = dc1.cppl.com
+		admin_server = dc1.cppl.com
+	}
+	ATHENA.MIT.EDU = {
```

Comment:
Please remove any configuration items, which are not relevant for the docker image usage